frame/support/src/weights.rs

```diff
@@ -153,6 +168,8 @@ pub enum SimpleDispatchInfo {
 	FixedOperational(Weight),
 	/// An operational dispatch with the maximum weight.
 	MaxOperational,
+	/// An mandatory dispatch with fixed weight.
```

Comment:
Is it correct to assume that `Mandatory` operations are treated like `Operational` variants with respect to priority? Should this be explicitly stated in the comment above?

Reply:
Updated docs to state that mandatory dispatches cannot be called directly from a transaction, so priority &c is moot.